docs/installation/server-config/security.rst

```diff
@@ -1,4 +1,227 @@
+    # r1.great-app.confluent.aws.prod.cloud:9093,r2.great-app.confluent.aws.prod.cloud:9094
+    bootstrap.servers=<bootstrap-server-endpoints>
+    ksql.sink.replicas=3
+    replication.factor=3
+    security.protocol=SASL_SSL
+    sasl.mechanism=PLAIN
+    sasl.jaas.config=\
+        org.apache.kafka.common.security.plain.PlainLoginModule required \
+        username="<confluent cloud access key>" \
+        password="<confluent cloud secret>";
+
+**Communicate with the Confluent Schema Registry over HTTPS**
+
+#. Specify the HTTPS endpoint in the ``ksql.schema.registry.url`` setting in the
+   KSQL configuration file:
+
+   .. code:: bash
+
+        ksql.schema.registry.url=https://<host-name-of-schema-registry>:<ssl-port>
+
+#. If the Schema Registry's SSL certificate is not signed by a CA that is recognized by the JVM by default, then you
+   must provide a suitable truststore via the ``KSQL_OPTS`` environment variable:
+
+   .. code:: bash
+
+       $ export KSQL_OPTS="-Djavax.net.ssl.trustStore=<path-to-trust-store>
+                           -Djavax.net.ssl.trustStorePassword=<store-password>"
+
+   or on the commandline when starting KSQL:
+
+   .. code:: bash
+
+      $ KSQL_OPTS="-Djavax.net.ssl.trustStore=<path-to-trust-store> -Djavax.net.ssl.trustStorePassword=<store-password>" ksql-server-start <props>
+
+The exact settings you need depend on what SASL mechanism your Kafka cluster is using and how your SSL certificates
+are signed. For more information, see the `Security section of the Kafka documentation <http://kafka.apache.org/documentation.html#security>`__
+and the :ref:`security parameter reference <security-config-ref>`.
+
+
+Using KSQL with a Kafka Cluster Secured with ACLs
+-------------------------------------------------
+
+You can use KSQL with Apache Kafka clusters that are secured with ACLs. The behavior depends on whether the cluster is
+interactive or non-interactive.
+
+.. tip:: For more information about ACLs see :ref:`kafka_authorization` and for more information about interactive and
+         non-interactive queries, see :ref:`restrict-ksql-interactive`.
+
+-------------------------
+Interactive KSQL clusters
+-------------------------
+
+Kafka clusters that use ACLs to control access to resources require clients to authenticate as a particular user. Interactive
+KSQL clusters require that this authenticated user has open access to create, read, write, delete topics, and use any
+consumer group.
+
+The required ACLs are:
+
+- ``DESCRIBE_CONFIGS`` permission on the ``CLUSTER``.
+- ``CREATE`` permission on the ``CLUSTER``.
+- ``DESCRIBE``, ``READ``, ``WRITE`` and ``DELETE`` permissions on the ``<any>`` ``TOPIC``.
+- ``DESCRIBE`` and ``READ`` permissions  on the ``<any>`` ``GROUP``.
+
+It is still possible to restrict the KSQL user from accessing specific resources using ``DENY`` ACLs. For example, you can add a
```

Comment:
> to restrict the _ksql-user_ from

Reply:
changed to `authenticated KSQL user` globally